integrations/cdi/narayana-jta-cdi/src/main/java/io/helidon/integrations/narayana/jta/cdi/NarayanaExtension.java

```diff
@@ -0,0 +1,132 @@
+/*
+ * Copyright (c) 2019 Oracle and/or its affiliates. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.helidon.integrations.narayana.jta.cdi;
+
+import java.util.Collection;
+
+import javax.enterprise.context.Dependent;
+import javax.enterprise.event.Observes;
+import javax.enterprise.inject.Any;
+import javax.enterprise.inject.CreationException;
+import javax.enterprise.inject.Default;
+import javax.enterprise.inject.spi.AfterBeanDiscovery;
+import javax.enterprise.inject.spi.Bean;
+import javax.enterprise.inject.spi.BeanManager;
+import javax.enterprise.inject.spi.CDI;
+import javax.enterprise.inject.spi.Extension;
+import javax.inject.Singleton;
+import javax.transaction.SystemException;
+import javax.transaction.Transaction;
+import javax.transaction.TransactionManager;
+import javax.transaction.TransactionScoped;
+import javax.transaction.UserTransaction;
+
+import com.arjuna.ats.jta.common.JTAEnvironmentBean;
+import com.arjuna.common.internal.util.propertyservice.BeanPopulator;
+
+/**
+ * A <a
+ * href="http://docs.jboss.org/cdi/spec/2.0/cdi-spec.html#spi">CDI 2.0
+ * portable extension</a> that adapts the <a
+ * href="https://narayana.io/">Narayana transaction engine</a> to a <a
+ * href="http://docs.jboss.org/cdi/spec/2.0/cdi-spec.html#part_2">CDI
+ * 2.0 SE environment</a>.
+ *
+ * @author <a href="mailto:laird.nelson@oracle.com">Laird Nelson</a>
+ */
+public final class NarayanaExtension implements Extension {
+
+
+  /*
```

Comment:
This file seems to be indented with 2 spaces instead of 4.

Reply:
Done.